xds/src/main/java/io/grpc/xds/client/XdsClientImpl.java

```diff
@@ -88,41 +84,45 @@ public void uncaughtException(Thread t, Throwable e) {
         }
       });
 
-  private final Map<ServerInfo, ControlPlaneClient> serverChannelMap = new HashMap<>();
+  protected final Map<Bootstrapper.ServerInfo, LoadStatsManager2> loadStatsManagerMap =
```

Comment:
The class is importing `Bootstrapper.ServerInfo`, yet there's a lot of changes from `ServerInfo` to `Bootstrapper.ServerInfo`.

Reply:
Previously there was an import of Bootstrapper.ServerInfo, so this wasn't really a change.  I've restored the import and gotten rid of the `Bootstrapper` qualifier.